### PR TITLE
Feature/rcspf 129399 tidy up api

### PR DESCRIPTION
**Pull request by ruadrian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,11 @@ This project is provided under the [Eclipse Public License 2.0](LICENSE)
 ### Set up the project in eclipse
 Import repository clone root folder as "Existing Projects into Workspace" and select "Search for nested projects" before clicking "Finish".
 
+#### Lombok
+We use Lombok to generate boilerplate. Lombok is compatible with various IDEs, find the setup instructions here [Lombok Eclipse Setup](https://projectlombok.org/setup/eclipse).
```
Comment (iachelini) [inline README.md:26]: The word "code" is missing, isn't it?

**Review by iachelini — APPROVED** [review]
I think its a good change to remove this "Sci" prefix.

File: README.md (hunk 1 above)
Comment (ruadrian) [reply README.md:26]: If you wish I can add 'code' but from what I know, boilerplate and boilerplate code are used interchangeably.

**Review by iachelini — APPROVED** [review]

**Comment by ruadrian** [comment]
Rebased

**Review by iachelini — APPROVED** [review]

**Pull request merged**